Gemfile

```diff
@@ -16,6 +16,12 @@ if ENV.key?('PUPPET_VERSION')
   puppet_version = "= #{ENV['PUPPET_VERSION']}"
 end
 
+gem 'beaker',
+    :git => 'https://github.com/petems/beaker.git',
+    :branch => 'MAINT-non_cygwin_windows_fixes',
+    :require => false
+gem 'beaker-rspec',                                                            :require => false
+
```

Comment:
Beaker works on Windows now? If not, we'll need to restrict this down to just ruby